Gopkg.toml

```diff
@@ -20,18 +20,24 @@
   name = "google.golang.org/grpc"
   version = "1.9.2"
 
-[[constraint]]
+[[override]]
+  name = "github.com/json-iterator/go"
+  version = "1.1.4"
+
+// TODO: use [[constraint]] when kubernetes-csi/kubernetes-csi-migration-library uses 1.13 code
```

Comment:
you can pull in the newest version of migration library now

Reply:
I think it does not make much sense to update for 1.13 beta, I'd wait until 1.13 is released and then update everything.